### PR TITLE
[pairing] Sync chat removing

### PR DESCRIPTION
**Pull request by rasom**
needed for status-im/status-react#12520

status: ready

### CODE REVIEW

**Review by Samyoul — APPROVED** [review]
Hey @rasom this looks great. The only thing that we seem to be missing are some tests that check that the functionality doesn't break in the future.

I've got some syncing tests https://github.com/status-im/status-go/blob/89251e841655082076ef304791b9ee3145f337c2/protocol/communities_messenger_test.go#L1091 that you can use as a guide

**Comment by rasom** [comment]
Sure, @Samyoul , I will add a test. Thanks.

**Comment by rasom** [comment]
@Samyoul is this enough?

**Comment by Samyoul** [comment]
Hey @rasom, that looks really good. Thank you.

File: protocol/messenger_installations_test.go [hunk 1]
```diff
@@ -216,17 +226,25 @@ func (s *MessengerInstallationSuite) TestSyncInstallation() {
 	s.Require().NoError(err)
 
 	var statusChat *Chat
+	var removedChat *Chat
 	for _, c := range allChats {
+		s.logger.Info("CHAT", zap.Any("ID", c.ID))
```
Comment (cammellos) [inline protocol/messenger_installations_test.go]: this could be removed

Comment (rasom) [reply protocol/messenger_installations_test.go]: done

**Pull request merged**